### ISSUE TITLE
Inquiry (Non-Issue)- Bibliography/References

### ISSUE THREAD

**Issue by edaggett17** [simbrain/simbrain#50]
Hello All,

Is there a repository or bibliography housing documentation of the source of the various functions utilized throughout SimBrain? For instance, if I wanted to investigate the origins of the "Decay" activation function within the neuron editor, where might I find that? 

Thank you,

Eben

**Comment by jyoshimi** [comment]
Not really, beyond comments here or there in the javadocs and documentation.   For decay functions I think Scott Hotton and I just thought about it.  His background is in math and dynamical systems, and what we did is similar to an exponential growth model: https://en.wikipedia.org/wiki/Exponential_growth.  In fact I believe what we can "relative decay" just is the difference equation on that page.

**Comment by edaggett17** [comment]
Cool! You all must have had a blast developing this program! As always, your work is much appreciated!
